api/public/cerbos/response/v1/response.proto

```diff
@@ -445,6 +445,17 @@ message InspectPoliciesResponse {
     json_schema: {description: "Inspect policies response"}
   };
 
+  message Attribute {
+    enum Type {
```

Comment:
Let's call this `kind` or `source`. We may want to add the actual data type at some point so let's keep `type` free for that.